Simple Inventory Management System/Simple Inventory Management System/InputHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿
+
+namespace SimpleInventoryManagementSystem
+{
+    public static class InputHelper
+    {
+        public static string ReadString(string str)
+        {
+            Console.Write(str);
+            return Console.ReadLine();
+        }
+
+        public static decimal ReadDecimal(string str, decimal minValue = 1)
```

Comment:
this is not generic enough, what if i need to read any decimal, minValue maybe needs to have decimal.MinValue, same for the other methods